assets/sass/protocol/base/elements/_forms.scss

```diff
@@ -42,79 +49,73 @@ textarea {
 }
 
 button,
-[type="button"],
-[type="reset"],
-[type="submit"] {
-    -webkit-appearance: button;
+[type='button'],
+[type='reset'],
+[type='submit'] {
+    appearance: button;
 }
 
 // Remove the default vertical scrollbar in IE 10+.
 textarea {
     overflow: auto;
+    height: auto;
 }
 
 // 1. Correct the odd appearance in Chrome and Safari.
 // 2. Correct the outline style in Safari.
-[type="search"] {
-    -webkit-appearance: textfield; // 1
+[type='search'] {
+    appearance: textfield; // 1
     outline-offset: -2px; // 2
 }
 
 // Remove the inner padding in Chrome and Safari on macOS.
-[type="search"]::-webkit-search-decoration {
-    -webkit-appearance: none;
+[type='search']::-webkit-search-decoration {
+    appearance: none;
 }
 
 // 1. Correct the inability to style clickable types in iOS and Safari.
 // 2. Change font properties to `inherit` in Safari.
 ::-webkit-file-upload-button {
-    -webkit-appearance: button; // 1
+    appearance: button; // 1
```

Comment:
@katiechurchwell basically anwhere we see `-webkit-appearance` we should update with a ignore comment on the line previous